docs/reference/ilm/actions/ilm-rollover.asciidoc

```diff
@@ -0,0 +1,196 @@
+[[ilm-rollover]]
+=== Rollover
+
+Phases allowed: hot.
+
+[WARNING]
+index format must match pattern '^.*-\\d+$', for example (`logs-000001`).
+[WARNING]
+The managed index must set `index.lifecycle.rollover_alias` as the
+alias to rollover. The index must also be the write index for the alias.
+
+[IMPORTANT]
+If a policy using the Rollover action is used on a <<ccr-put-follow,follower
+index>>, policy execution will wait until the leader index rolls over (or has
+<<skipping-rollover, otherwise been marked as complete>>), then convert the
+follower index into a regular index as if <<ilm-unfollow-action,the Unfollow
+action>> had been used instead of rolling over.
+
+For example, if an index to be managed has an alias `my_data`. The managed
+index "my_index-000001" must be the write index for the alias. For more information, read
+<<indices-rollover-is-write-index,Write Index Alias Behavior>>.
+
+[source,console]
+--------------------------------------------------
+PUT my_index-000001
+{
+  "settings": {
+    "index.lifecycle.name": "my_policy",
+    "index.lifecycle.rollover_alias": "my_data"
+  },
+  "aliases": {
+    "my_data": {
+      "is_write_index": true
+    }
+  }
+}
+--------------------------------------------------
```

Comment:
This isn't properly blocked off, which causes the second para and snippet to display outside the block. I'd move it after the description.
```suggestion
[IMPORTANT]
--
If a policy using the Rollover action is used on a <<ccr-put-follow,follower
index>>, policy execution will wait until the leader index rolls over (or has
<<skipping-rollover, otherwise been marked as complete>>), then convert the
follower index into a regular index as if <<ilm-unfollow-action,the Unfollow
action>> had been used instead of rolling over.

For example, if an index to be managed has an alias `my_data`. The managed
index "my_index-000001" must be the write index for the alias. For more information, read
<<indices-rollover-is-write-index,Write Index Alias Behavior>>.

[source,console]
--------------------------------------------------
PUT my_index-000001
{
  "settings": {
    "index.lifecycle.name": "my_policy",
    "index.lifecycle.rollover_alias": "my_data"
  },
  "aliases": {
    "my_data": {
      "is_write_index": true
    }
  }
}
--------------------------------------------------
--
```